lib/jsonapi/view.ex

```diff
@@ -59,6 +59,17 @@ defmodule JSONAPI.View do
   If you always want to include a relationship. First make sure its always preloaded
   and then use the `[user: {UserView, :include}]` syntax in your `includes` function. This tells
   the serializer to *always* include if its loaded.
+
+  There are several options available through configuration. These include switching from underscores
```

Comment:
Do we need to copy + paste the README doc into here?  I think we can do without.